modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -155,9 +152,7 @@ class Form extends Component {
 					this.element
 				);
 
-				this._eventHandler.add(
-					this._autoSave.on('autosaved', this._updateAutoSaveMessage)
-				);
+				this._autoSave.on('autosaved', this._updateAutoSaveMessage);
```

Comment:
Just for auditing purposes, [this is the implementation of `AutoSave`](https://github.com/liferay-frontend/liferay-portal/blob/ef3ae75e32fb38f4bebb6987b6048ac98bdf43c6/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/util/AutoSave.es.js); looks like when we `dispose()` it, it will call `stop()` which will stop `save()` from getting called, which means the event will never be emitted. So 👍

Reply:
Yes, I figured out that it was self-reliant so left it to it's own devices
![](https://media1.giphy.com/media/2rqEdFfkMzXmo/giphy.gif)